src/test/java/dev/group4/services/UserServiceTests.java

```diff
@@ -0,0 +1,34 @@
+package dev.group4.services;
+
+import dev.group4.entities.User;
+import dev.group4.repos.UserRepo;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class UserServiceTests {
+    private User testUser;
+    @Autowired
+    private UserRepo userRepo;
+    @Order(1)
```

Comment:
Ideally we should try not to use test orderer. not sure if it requires the TestOrderer annotation thing in Java if we are going to stay using it though

Reply:
ordering tests is a bad thing? figured it just lets us not have to reinitialize a test object for every single test

Reply:
I'm not sure, at least when I talked to my brother on the whole test orderer he said most of the time each unit test operates completely independently at the place he works. I try to ask him a little about the curriculum from time to time. Also in my last group we had a couple people run into issues with the test orderer in either maven or git actions idr. I think they found out it wasn't the orderer in the end but I'm not sure.